modules/machineset-yaml-aws.adoc

```diff
@@ -2,22 +2,28 @@
 //
 // * machine_management/creating-infrastructure-machinesets.adoc
 // * machine_management/creating_machinesets/creating-machineset-aws.adoc
+// * post_installation_configuration/aws-compute-edge-tasks.adoc
 
 ifeval::["{context}" == "creating-infrastructure-machinesets"]
 :infra:
 endif::[]
+ifeval::["{context}" == "aws-compute-edge-tasks"]
+:edge:
+endif::[]
 
 :_content-type: REFERENCE
 [id="machineset-yaml-aws_{context}"]
 =  Sample YAML for a compute machine set custom resource on AWS
 
 This sample YAML defines a compute machine set that runs in the `us-east-1a` Amazon Web Services (AWS) zone and creates nodes that are labeled with
```

Comment:
We'll need to add ifdef here for edge compute pool. The `us-east-1a` is not a valid Local Zone Name, furthermore, the "sample snippet" does not set the zone name in `availabilityZone`.

Reply:
Thanks. I see the following in the snippet, so does this resolve your `availabilityZone` query?

![Screenshot from 2023-10-17 09-43-49](https://github.com/openshift/openshift-docs/assets/57954076/c05511af-560c-40b0-8313-dd882b2736e6)